README.md

```diff
@@ -0,0 +1,22 @@
+#Project ToolShop
+
+User can buy tools for repairing
+
+#How install a project?
+
+1.Install VS Code
+2.npm init playwright@latest  --> install playwright latest version
+   Choose TypeScript
+3.npx playwright test  --> run a test
+4.Add extension "Playwright Test for VS Code"
+5.Add a script to package.json 
```

Comment:
this command should be added by yourself so that anyone who will clone the repo can use it.